### PR TITLE
Renamed fixed_process to physics_process

### PR DESCRIPTION
**Pull request by AndreaCatania**
Related to this godotengine/godot#11659

### CODE REVIEW

**Comment by AndreaCatania** [comment]
cc @NathanLovato

File: learning/step_by_step/scripting_continued.rst [hunk 1]
```diff
@@ -29,9 +29,13 @@ frames per second (FPS) of the application:
 The delta parameter describes the time elapsed (in seconds, as
 floating point) since the previous call to "_process()".
 
-Fixed processing is similar, but only needed for synchronization with
-the physics engine. It is called a fixed amount of times per second
-(defined in the Project Settings, but 60 by default).
+Physics processing is similar, but should be used for all the processes that
```
Comment (NathanLovato) [inline learning/step_by_step/scripting_continued.rst]: Reworked, closer to the [writing guidelines](docs.godotengine.org/en/latest/community/contributing/docs_writing_guidelines.html)
`Physics processing is similar, but it should be used for all the processes that must happen before each physics step. For example, to move a character.
It is always runs before a physics step and it is called at fixed time intervals, 60 times per second by default. Change the value in the Project Settings.`

**Comment by AndreaCatania** [comment]
@NathanLovato I did it

**Comment by NathanLovato** [comment]
A few typos remaining, but let's merge it - the article needs editing anyway

**Comment by NathanLovato** [comment]
Thanks for your work!

**Pull request merged**